sonar-kotlin-plugin/src/test/java/org/sonarsource/kotlin/plugin/KotlinProfileDefinitionTest.kt

```diff
@@ -19,19 +19,30 @@
  */
 package org.sonarsource.kotlin.plugin
 
-import org.assertj.core.api.Assertions
+import com.google.gson.Gson
+import com.google.gson.JsonObject
+import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
 import org.sonar.api.server.profile.BuiltInQualityProfilesDefinition
 
 internal class KotlinProfileDefinitionTest {
 
+    @Test
+    fun `Sonar_way_profile is a valid json file`() {
+        val classLoader = KotlinProfileDefinition::class.java.classLoader
+        val jsonContent = classLoader.getResource(KotlinProfileDefinition.PATH_TO_JSON)?.readText()
+        val json = Gson().fromJson(jsonContent, JsonObject::class.java)
+        assertThat(json.get("name").asString).isEqualTo("Sonar way")
+        assertThat(json.get("ruleKeys").asJsonArray).hasSizeGreaterThan(80)
```

Comment:
Maybe because we use 80 and "Sonar way" in both tests, we should declare a constant for it, so we keep both tests in sync automatically?